lib/errors.coffee

```diff
@@ -31,7 +42,10 @@ exports.handle = (error) ->
 
 	patterns.printErrorMessage(message)
 
-	captureException(error)
+	Promise.fromCallback((cb) ->
+		console.log('inside promise')
+		analytics.captureException(error)
+	)
```

Comment:
This promise never resolves: you never call `cb`, so this _always_ waits the second-long timeout below, which isn't good.

Reply:
I have changed that. Although the .finally at the end behaved weirdly.